maven-plugin/src/main/java/org/sonatype/nexus/api/common/ArgumentUtils.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.api.common;
```

Comment:
all the classe in org.sonatype.nexus.api are a copy from https://github.com/sonatype/nexus-java-api/tree/main/api, So I'm reusing the client, one interesting discrepancy is that here the package starts with `org` instead of `com`